src/main/resources/i18n/Resources_es.properties

```diff
@@ -1,16 +1,8 @@
 # suppress inspection "UnusedProperty" for whole file
-#TODO translate
-loginDialogLabelUsingCustomCredentials=Using own Google API secret (safety warnings expected)
-#TODO translate
-loginDialogLabelUsingDefaultCredentials=You can use own Google API secret to increase limits
-#TODO translate
-loginDialogConfigureButtonText=Configure...
-googleStorageWarning=Todos los archivos multimedia subidos utilizara calidad ORIGINAL y sumaran al almacenamiento de su cuenta de Google.
-# this is a tooltip that pops up when Google API quota is exceeded; the app then slows down by applying exponential backoff; the backoff delay in seconds
-# is appended to the phrase below
-#TODO translate, the English version now says:
-#backoffNotice=Google currently imposes a limit on this activity. This is normal, but sometimes can lead to minutes or even hours before any progress \
-#  can be made. Please be patient, stop the upload and try later or switch to using your own Google API Client Secret in settings. Latest delay in seconds:
+loginDialogLabelUsingCustomCredentials=Usar su Google API secret (Es de esperar que aparezcan advertencias de seguridad)
+loginDialogLabelUsingDefaultCredentials=Puede usar su Google API secret para aumentar los limites
+loginDialogConfigureButtonText=Configurar...
+googleStorageWarning=Todos los archivos multimedia subidos utilizaran calidad ORIGINAL y sumaran al almacenamiento de su cuenta personal de Google.
 backoffNotice=Google actualmente impone un límite a esta actividad. Esto es normal, pero a veces puede llevar minutos o incluso horas antes de que cualquier \
```

Comment:
This notice has changed - see the TODO that was there. Can you translate and update it please? change

Reply:
More specifically, at the end of the first sentence I added “ or switch to using your own Google API Client Secret in settings”.

Reply:
I meant to add correct this long sentence: "Tenga paciencia o detenga la carga e intentelo más tarde." Previously, it meant "Please be patient or stop the upload and try later". However, now it should read: "Please be patient, stop the upload and try later or switch to using your own Google API Client Secret in settings". Thanks!